jf_agent/jf_jira/jira_download.py

```diff
@@ -41,9 +41,7 @@ def download_users(
     # searching a different way
     if 950 <= len(jira_users) <= 1000:
         logger.info(
-            level=logging.INFO,
-            msg=f'Page limit reached with {len(jira_users)} users, '
-            'falling back to search by letter method.',
+            f'Page limit reached with {len(jira_users)} users, falling back to search by letter method.'
```

Comment:
This was the same bug that I found in `ratelimit.py`, but somewhere else. We haven't hit here yet, thankfully, but this shouldn't error our like it did for the ratelimit